CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ enable_testing()
 
 find_package(Threads REQUIRED)
 find_package(LLVM    REQUIRED CONFIG)
-find_package(Boost   REQUIRED)
+find_package(Boost   REQUIRED COMPONENTS thread filesystem system program_options log regex)
```

Comment:
In case the custom function works without manually specifying the `COMPONENTS [...]` here, I'd rather have these removed, as this just introduces an extra place where stuff needs to be maintained.

Reply:
Uh-huh. Okay, I understand it now. Weird either way, I believe it was more in line with common sense that if no `COMPONENTS` is given, then it means "every component" and thus I could fetch what is installed on the system...